Dockerfile

```
@@ -126,6 +130,14 @@ RUN git clone -b master https://github.com/jesims/circleci-tools.git \
 ENV PATH=$PATH:/tmp/circleci-tools/
 RUN node -v > .node_version
 
+#-- Install cljog
+RUN wget https://raw.githubusercontent.com/axrs/cljog/${CLJOG_VERSION}/cljog \
+  && chmod u+x cljog \
+  && mv cljog /usr/bin/ \
+  && wget https://raw.githubusercontent.com/axrs/cljog/${CLJOG_VERSION}/example-scripts/echo.clj \
+  && chmod u+x echo.clj \
+  && ./echo.clj
```

then remove ./echo.clj
```suggestion
  && ./echo.clj && rm echo.clj
```

---

👍